src/Evolution/EventsAndDenseTriggers/DenseTriggers/Filter.hpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <pup.h>
+#include <pup_stl.h>
+#include <string>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Evolution/EventsAndDenseTriggers/DenseTrigger.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Trigger.hpp"
+#include "Utilities/Registration.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace DenseTriggers {
+template <typename TriggerRegistrars, typename DenseTriggerRegistrars>
+class Filter;
+
+namespace Registrars {
+template <typename TriggerRegistrars>
+using Filter =
+    Registration::Registrar<DenseTriggers::Filter, TriggerRegistrars>;
+}  // namespace Registrars
+
+/// \ingroup EventsAndTriggersGroup
+/// Filter activations of a dense trigger using a non-dense trigger.
```

Comment:
I'm not sure I really understand when I should use this trigger. Is there a basic example or two that would make sense to add to the dox?